src/jselfbot/commands/BashCmd.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2017 Artu
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package jselfbot.commands;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Collection;
+import jselfbot.Command;
+import jselfbot.entities.Emoji;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.events.message.MessageReceivedEvent;
+
+/**
+ * Runs a shell command.
+ * @author Spotlight
+ * @author Artu
+ */
+public class BashCmd extends Command {
+    public BashCmd() {
+        this.name = "bash";
+        this.description = "run a command on a shell";
+    }
+
+    @Override
+    protected void execute(String args, MessageReceivedEvent event) {
+        if(args.isEmpty()) {
+            reply("Cannot execute a empty command!", event);
+            return;
+        }
+
+        StringBuilder output = new StringBuilder();
+        String finalOutput;
+        try {
+            Process p = Runtime.getRuntime().exec(args);
+            p.waitFor();
+            BufferedReader reader =
+                    new BufferedReader(new InputStreamReader(p.getInputStream()));
+
+            String runningLineOutput;
+            while ((runningLineOutput = reader.readLine())!= null) {
+                output.append(runningLineOutput).append("\n");
+            }
+
+            if (output.toString().isEmpty()) {
+                reply("Done, with no output!", event);
+                return;
+            }
+
+            // Remove linebreak
+            finalOutput = output.substring(0, output.length() - 1);
+        } catch (IOException e) {
+            reply("I wasn't able to find the command `" + args + "`!", event);
+            return;
+        } catch (Exception e) {
+            e.printStackTrace();
+            reply("An unknown error occurred! Check the bot console.", event);
+            return;
+        }
+
+        // Actually send
+        try {
+            reply("Input: ```\n" + args + "``` Output: \n```\n" + finalOutput + "```", event);
+        } catch (IllegalArgumentException e) {
+            reply("Command output too long!", event);
```

Comment:
Can you make it so it prints it to console, similar to how it does on an exception?